src/index.html

```diff
@@ -1,15 +1,132 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form>
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <div class="input">
+          <label for="surname">Surname:
+            <input id="surname" type="text">
+          </label>
+        </div>
+
+        <div class="input">
+          <label for="name">Name:
+            <input id="name" type="text"
+            maxlength="30">
+          </label>
+        </div>
+
+        <div class="input">
+          <label for="age" class="input">How old are You?
+            <input id="age" type="number"
+            value="12" min="12"
+            max="100">
```

Comment:
```suggestion
            <input
              id="age"
              type="number"
              value="12"
              min="12"
              max="100"
            >
```
each attribute on a new line